tools/update_deleted_packages.sh

```diff
@@ -64,6 +65,10 @@ while (("$#")); do
       bazelrc_path="${2}"
       shift 2
       ;;
+    "--exclude-dir")
```

Comment:
Can we please make the flag be `--exclude_dir`? Other tools use underscore for their flags.